src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser.md

```diff
@@ -86,9 +86,17 @@ $ k6 run script.js
 ```
 
 ```bash
-# When using the `k6:master-with-browser` Docker image, you need to add `--cap-add=SYS_ADMIN`
-# to grant further system permissions on the host for the Docker container.
-docker run --rm -i --cap-add=SYS_ADMIN grafana/k6:master-with-browser run - <script.js
+# WARNING!
+# Be aware that grafana/k6:master-with-browser launches Chrome browser by setting the
+# 'no-sandbox' argument. Use it only with trustworthy websites.
```

Comment:
```suggestion
# The grafana/k6:master-with-browser image launches a Chrome browser by setting the
# 'no-sandbox' argument. Only use it with trustworthy websites.
```

Some small nits. You could also reword it as:

```
When using the grafana/k6:master-with-browser image, the container launches a Chrome browser with the 'no-sandbox' argument. Only use it with trustworthy websites. 
```

Reply:
Changed in [7b98b68](https://github.com/grafana/k6-docs/pull/1330/commits/7b98b68942a04393fb1e2f587c2e1f0d570d61e3).

Reply:
@heitortsergent 

Doesn't _"only use it"_ mean you **prefer** to use it with trustworthy websites, but it's OK to use it on untrusted sites? As the "only" modifies the "use" part now instead of "it". 🤔

The previous "only with trustworthy websites" is more restrictive, implying that you should avoid untrustworthy websites altogether when using "it,".

I don't have a strong opinion :)

Reply:
My understanding, and reading around, is that both sentences have the exact same meaning. Personally I feel the sentence starting with `Only` sounds better, so merging it as it is to also not delay the PR review process anymore, as the image is already public in Docker registry. Thanks for your feedback guys!